net-mgmt/zabbix-agent/Makefile

```diff
@@ -1,7 +1,12 @@
 PLUGIN_NAME=		zabbix-agent
-PLUGIN_VERSION=		1.9
+PLUGIN_VERSION=		1.10
 PLUGIN_COMMENT=		Zabbix monitoring agent
-PLUGIN_DEPENDS=		zabbix5-agent
 PLUGIN_MAINTAINER=	opnsense@moov.de
+PLUGIN_VARIANTS=	zabbix5 zabbix54
 
+zabbix5_NAME=		zabbix5-agent
```

Comment:
@fraenki please take note that this moves the plugin name from zabbix-agent to zabbix5-agent and zabbix-agent will show up as orphaned. For interoperability it might be better to call this zabbix-agent for now as it is the same port?

Reply:
@fichtner Oh yes, indeed. I've addressed this in https://github.com/opnsense/plugins/pull/2656/commits/137de8ee8718344d2610f6562088db1e58229f01.